examples/disp_s1_sample_runconfig-v3.0.0-er.1.0.yaml

```diff
@@ -0,0 +1,756 @@
+# Sample RunConfig for use with the DISP-S1 PGE
+# This RunConfig should require minimal changes in order to be used with the
+# OPERA PCM.
+
+RunConfig:
+    # Name for the RunConfig, may be any string
+    Name: OPERA-DISP-S1-PGE-SAMPLE-CONFIG
+
+    Groups:
+        # PGE-specific RunConfig section
+        # This section is only used by the PGE, however, paths to inputs/outputs
+        # should align with the similar sections of the SAS RunConfig
+        PGE:
+            PGENameGroup:
+                PGEName: DISP_S1_PGE
+
+            InputFilesGroup:
+                InputFilePaths:
+                    - disp_s1_pge_test/input_dir
+
+            DynamicAncillaryFilesGroup:
+                AncillaryFileMap:
+                    dem_file: disp_s1_pge_test/input_dir/dem_4326.tiff
+
+            ProductPathGroup:
+                OutputProductPath: disp_s1_pge_test/output_dir
+                ScratchPath: disp_s1_pge_test/scratch_dir
+
+            PrimaryExecutable:
+                ProductIdentifier: DISP_S1
+                ProgramPath: /bin/echo
+                ProgramOptions:
+                    - 'hello world > disp_s1_pge_test/output_dir/disp_s1_test.txt;'
+                    - '/bin/echo DISP-S1 invoked with RunConfig'
+                ErrorCodeBase: 500000
+                SchemaPath: pge/disp_s1/schema/disp_s1_sas_schema.yaml
+                IsoTemplatePath:
+
+            QAExecutable:
+                Enabled: False
+                ProgramPath:
+                ProgramOptions: []
+
+            DebugLevelGroup:
+                DebugSwitch: False
+                ExecuteViaShell: True  # Must be set to True for test to work
```

Comment:
This should be set to False and the comment removed for the "example" runconfigs

Reply:
Done